build-nuxt-site.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
```

Comment:
Every bash script should start with `set -euo pipefail` so that errors are propagated the way you expect they should. In this case probably this script is failing because the nodejs dependencies have not been installed.